release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -742,6 +742,19 @@ On the hub cluster, you can use `FirmwareSchema` CRs to discover managed cluster
 
 For more information, see xref:../edge_computing/ztp-deploying-far-edge-sites.adoc#ztp-configuring-host-firmware-with-gitops-ztp_ztp-deploying-far-edge-sites[Managing host firmware settings with {ztp}].
 
+[id="ocp-4-17-image-based-upgrade-enhancements_{context}"]
+==== Image-based upgrade enhancements
+
+With this release, the image-based upgrade introduces the following enhancements:
+
+* Simplifies the upgrade process for a large group of managed clusters by adding the `ImageBasedGroupUpgrade` API on the hub
+* Labels the managed clusters for action completion when using the `ImageBasedGroupUpgrade` API
+* Improves seed cluster validation before the seed image generation
+* Automatically cleans up the container storage disk if usage reaches a certain threshold on the managed clusters
+* Adds comprehensive event history in the new `status.history` field of the `ImageBasedUpgrade` CR
```

Comment:
Won't hold up the merge over this but this should say "custom resource" since it's the first mention of the term in this section:
```suggestion
* Adds comprehensive event history in the new `status.history` field of the `ImageBasedUpgrade` custom resource
```